.travis.yml

```diff
@@ -16,11 +16,15 @@ matrix:
     - php: 7.2
       env:
         - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction"
+        - APP_DEBUG=1
+        - APP_ENV=test
     - php: 7.3
       env:
         - COMPOSER_FLAGS="--prefer-dist --no-interaction"
         - CODE_COVERAGE="--coverage-clover=coverage.clover"
         - PHPSTAN=true
+        - APP_DEBUG=1
+        - APP_ENV=test
```

Comment:
APP_DEBUG should not be needed

APP_ENV can be set on [global](https://github.com/sulu/SuluCommunityBundle/blob/381e6c641c082d9a98848f610532b4be98081f6e/.travis.yml#L10-L12) but should also be force in [phpunit.xml.dist](https://github.com/sulu/SuluCommunityBundle/blob/381e6c641c082d9a98848f610532b4be98081f6e/phpunit.xml.dist#L21-L23)